src/test/java/io/jenkins/plugins/opentelemetry/JenkinsOpenTelemetryPluginConfigurationIntegrationTest.java

```diff
@@ -0,0 +1,66 @@
+package io.jenkins.plugins.opentelemetry;
+
+import io.jenkins.plugins.opentelemetry.semconv.JenkinsMetrics;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.testing.exporter.InMemoryMetricExporterProvider;
+import io.opentelemetry.sdk.testing.exporter.InMemoryMetricExporterUtils;
+import io.opentelemetry.semconv.ServiceAttributes;
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Map;
+import java.util.Optional;
+import jenkins.model.Jenkins;
+import org.junit.After;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.is;
+
+@WithJenkins
+public class JenkinsOpenTelemetryPluginConfigurationIntegrationTest {
+    static {
+        OpenTelemetryConfiguration.TESTING_INMEMORY_MODE = true;
+    }
+
+    @BeforeEach() public void before() {
+        GlobalOpenTelemetry.resetForTest();
+    }
+
+    @After
+    public void after() throws Exception {
+        InMemoryMetricExporterProvider.LAST_CREATED_INSTANCE.reset();
+    }
+
+    @Test
+    @Issue("https://github.com/jenkinsci/opentelemetry-plugin/issues/1156")
+    public void configLoadReconfiguresOtelSdk(JenkinsRule r) throws Exception {
```

Comment:
This test is highly synthetic, so IDK if we even want to keep it, but it does demonstrate the issue, and fails without the fix.